RELEASE/scripts/autoscend/auto_king.ash

```diff
@@ -1,23 +1,23 @@
-script "kingsool.ash";
```

Comment:
moved this along with presool and postsool to the autoscend sub dir so only autoscend.ash is at the top level. seems cleaner. Everything still works but let me know if theres a reason they were at the top level in the first place.

Reply:
love it

Reply:
love it

Reply:
Nice one. This has been on my todo list since I started working on sl_ascend.